app/static/src/app/store/basic/basic.ts

```diff
@@ -3,16 +3,15 @@ import { actions } from "./actions";
 import { mutations } from "./mutations";
 import { errors } from "../errors/errors";
 import { BasicState } from "./state";
-import { model, defaultState as defaultModelState } from "../model/model";
-import { code, defaultState as defaultCodeState } from "../code/code";
+import { model } from "../model/model";
+import { code } from "../code/code";
 
-const defaultState: () => BasicState = () => {
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const defaultState: () => any = () => {
     return {
         appType: "basic",
         appName: null,
-        config: null,
-        code: defaultCodeState,
-        model: defaultModelState
+        config: null
```

Comment:
This seemed like the least disruptive way to remove the browser warning about a module overwriting a state property - leave properties in the state (code and model) but don't provide a default to be overwritten (so avoid the warning) - but this means that TypeScript will error unless we make the return type here `any`.